test/system-script.test.js

```diff
@@ -1,133 +1,125 @@
 describe('system-amd-script', function() {
-  let validManifestUrl = 'http://localhost:' + window.location.port + '/base/test/manifests/simple.json';
-
-  let system;
+  let SystemJS;
 
   beforeEach(function() {
-    system = new System.constructor();
-    window.system = system;
-    window.SystemJS = system;
+    SystemJS = new System.constructor();
+    window.SystemJS = SystemJS;
     window.define = function () {
       window.canopyDefine.apply(window, arguments);
     }
+    const scripts = document.querySelectorAll('script[data-system-amd-name]')
+    for (let i=0; i<scripts.length; i++) {
+      scripts[i].remove() }
```

Comment:
this is necessary because we are resetting the scriptNameMap which makes SystemJS.import()'ing existing scripts fail since they depend on the scriptNameMap having entries for the existing scripts.